src/algorithms/tonal/tonicindianartmusic.cpp

```diff
@@ -198,14 +198,12 @@ void TonicIndianArtMusic::compute() {
   /*implementing the decision tree*/
 
   // Prevent segmentation fault
-  if (peak_locs.size() == 0) { 
+  if (peak_locs.size() == 0){ 
```

Comment:
This error message is not very informative to the user. Alternative:
> TonicIndianArtMusic: No predominant pitch salience peaks found in the specified tonic frequency region [minTonicFrequency, maxTonicFrequency]".

Reply:
updated